src/global/helpers/focus.ts

```diff
@@ -1,5 +1,24 @@
 export const IS_FOCUSABLE_QUERY = `:is(button, [href], input, select, textarea, details, summary:not(:disabled), [tabindex], sbb-button, sbb-link, sbb-menu-action):not([disabled]):not([tabindex="-1"]):not([static])`;
 
+export function getFocusableElements(elements: HTMLElement[]): HTMLElement[] {
```

Comment:
- add note about possible expensive call
- maybe compare to kagekiri implementation